### ISSUE TITLE
one progressbar for all posts, while still using pagination

### ISSUE THREAD

**Issue by opicron** [level-level/wp-cli-facetwp#5]
I could not stand the fact that I did not see an overall progress. So I made an adjustment so that your pagination is still working but the overall progress is shown in ONE progress bar.

And I am using the official index method as instructed by FacetWP instead of yours. Please see:
https://facetwp.com/how-to-trigger-the-indexer-programmatically/

Please see below improvement:

```
    function index( $args, $assoc_args ) {

        error_reporting(0);

        $post_type = 'any';

        if ( isset( $args[0] ) ) {
            $post_type = $args[0];
        }

        $posts_per_page = 100;
        $page = 1;

        $args = array(
                'posts_per_page'    => $posts_per_page,
                'paged'             => $page,
                'post_type'         => $post_type,
                'post_status'       => 'publish',
                'fields'            => 'ids',
                'orderby'           => 'ID',
                'cache_results'     => false,
        );

        $my_query = new WP_Query( $args );
        $total = $my_query->found_posts;

        $progress_bar = WP_CLI\Utils\make_progress_bar('Indexing', $total );

        do {
                if ($page != 1)
                {
                        $args['paged'] = $page;
                        $my_query = new WP_Query( $args );
                }

                foreach ( $my_query->posts as $key => $post_id )
                {
                        //WP_CLI::line( $post_id );
                        $progress_bar->tick();
                        FWP()->indexer->index( $post_id );

                }


            $page++;

            // Free up memory
            $this->stop_the_insanity();

        } while ( $my_query->have_posts() );

        $progress_bar->finish();

    }


```

**Comment by NielsdeBlaauw** [comment]
Hey @opicron 

Thanks for contributing! Good solution. If you can open a Pull Request I'd happily accept make this available for all users.

Otherwise let me known and I'll edit it myself, with correct attribution.

**Comment by opicron** [comment]
Please see pull request.

https://github.com/level-level/facetwp-wp-cli/pull/7

**Comment by NielsdeBlaauw** [comment]
Fixed in #7.

Thanks @opicron!
